internal/k8shandler/elasticsearch.go

```diff
@@ -100,3 +101,40 @@ func (er *ElasticsearchRequest) updatePrimaryShards() {
 		}
 	}
 }
+
+func (er *ElasticsearchRequest) isIndexBlocked(index string) bool {
```

Comment:
can we add a mocked test for this?

Reply:
Added for `isIndexBlocked` and `unblockIndex`